docs/responses.md

```
@@ -0,0 +1,434 @@
+  -H "Content-Type: application/json" \
+  -H "Authorization: Bearer $TOKEN" \
+  -d '{
+    "input": "Explain Kubernetes architecture",
+    "model": "openai/gpt-4-turbo",
+    "stream": true,
+    "store": true
+  }'
+```
+
+**Streaming Response (SSE format):**
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)